setup/bundles.rst

```diff
@@ -173,7 +173,7 @@ Before diving into the specifics of the most common edge cases, the general
 recommendation is to **not rely on the Symfony Kernel version** to decide which
 code to use::
 
-    if (Kernel::VERSION_ID <= 20800) {
+    if (Kernel::VERSION_ID < 20800) {
```

Comment:
What about simply using the `Kernel::MAJOR_VERSION` instead ?
The code sample does not need to be more accurate and is only here to demonstrate you should not rely on the kernel version to adapt code.